### PR TITLE
fix: use 'allow' rather than 'white' for allowlist

### PR DESCRIPTION
**Pull request by bnb**
Switches to use "allowlist" terminology rather than "blocklist" terminology

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
```
Comment (ljharb) [inline .vscode/settings.json:2]: what’s this for? generally editor things go in global/user config rather than in each repo

**Review by ljharb — APPROVED** [review]
Confirm no API a changes. LGTM pending comment.

**Comment by kemitchell** [comment]
@bnb do you want small things raised here in PR, or would you prefer me merge and then fettle as we like?

**Comment by kemitchell** [comment]
I'm thinking:
- [ ] `git rm` the VSCode config
- [ ] `s/allowlisted/allowed/g`
- [ ] proof comments for readability

File: .vscode/settings.json (hunk 1 above)
Comment (bnb) [reply .vscode/settings.json:2]: oop my bad, this was automatically added by VS Code and I didn't catch it

**Comment by bnb** [comment]
I did the first two, not sure what you mean by the last one @kemitchell 👀

**Comment by kemitchell** [comment]
@bnb you license your work under [Apache 2.0](https://github.com/jslicense/licensee.js/blob/main/LICENSE)? Any chance your employer owns this work, as done within the scope of your job?

This is mostly renaming a bunch of things, and may not fall under copyright. But this package does get used by some companies with money, so I want to make sure we stay clean on the IP side.

**Comment by bnb** [comment]
Wanted to give you an official answer with no question: yes, under Microsoft's  Small Contribution Exception, this is unquestionably allowed. I was informed of this by Microsoft's legal organization (CELA).

**Pull request merged**